packages/manager/apps/hub/src/dashboard/new-pricing/index.js

```diff
@@ -0,0 +1,21 @@
+import angular from 'angular';
+import '@uirouter/angularjs';
+import 'oclazyload';
+
+const moduleName = 'ovhManagerHubPricePageLazyLoading';
+
+angular.module(moduleName, ['ui.router', 'oc.lazyLoad']).config(
+  /* @ngInject */ ($stateProvider) => {
+    $stateProvider.state('app.dashboard.newPricing.**', {
+      url: 'new-prices',
```

Comment:
```suggestion
      url: '/new-prices',
```

Reply:
I tried adding `/new-prices` and it was not loading the page. so undo the changes.